src/olympia/pages/templates/pages/shield_study_11.html

```diff
@@ -2,58 +2,176 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <title>Plugin Safety</title>
```

Comment:
For whatever reasons (mostly because it will appear in the list of add-ons as "Plugin Safety"), we're leaving this not translated.